README.md

```diff
@@ -0,0 +1,22 @@
+# Additional Fruits
+This module adds various different fruit blocks and includes 3 different stages for each plant.
+
+**Types of Fruits:**
+* Blueberry
+* Bushes
+  - Cranberry
+  - Gojiberry
+  - Guava
+  - Peach
+* Gooseberry
+* Raspberry
+* Strawberry
+
+
+How to Contribute
+ ============
+ 1. Fork and clone this repository locally
+ 2. To add another **fruit block** or **fruit bush**, create a new prefab
+ 3. Add a texture to your fruit, a PNG file is reccomended
+ 4. After you have finished, submit a PR (pull request) to this repository 
```

Comment:
It may again be good to link to somewhere else that has more general instructions on how to fork, clone, build, commit, push, make PRs etc.